PDFOptimizer/Release_Notes.html

```diff
@@ -17,7 +17,27 @@
   <body>
         <h1>Release Notes, Datalogics PDF Optimizer</h1>
   
-	<div class="entry-content"><p><strong>Version 3.0</strong> (March 8, 2022)<br/>
+	<div class="entry-content">
+<p><strong>Version 3.2</strong> (May 17, 2023)<br/>
```

Comment:
the items here look like they correspond with the 3.1.1 release -
https://wiki-datalogics.atlassian.net/wiki/spaces/PROD/pages/7803213/PDF+Optimizer+Release+History

3.2 should mention PDFOPT-241 but doesn't have much else.

Reply:
the new feature of pdf/x options was PDFOPT-241 so it made sense to me to not put the jira issue ID there

since v3.1.1 wasn't a public release it seems less confusing to lump it under v3.2

Reply:
3.1.1 was released to the public on 23 Feb 2023 - according to https://wiki-datalogics.atlassian.net/wiki/spaces/PROD/pages/7803213/PDF+Optimizer+Release+History
is that not correct?

Reply:
ok, my mistake, should be all set now